test/WebSites/OutOfProcessWebSite/Startup.cs

```diff
@@ -130,5 +130,26 @@ private async Task ServerAddresses(HttpContext context)
         {
             await context.Response.WriteAsync(string.Join(",", _serverAddresses.Addresses));
         }
+
```

Comment:
Is it worth extracting common methods to a shared file between inproc and outofproc test site?

Reply:
Another PR; rather keep this change smaller than larger.